src/main/pg/gyrodev.h

```diff
@@ -45,6 +45,7 @@ typedef struct gyroDeviceConfig_s {
     ioTag_t extiTag;
     uint8_t alignment;        // sensor_align_e
     sensorAlignment_t customAlignment;
+    ioTag_t clkIn;
 } gyroDeviceConfig_t;
```

Comment:
You’ll need to bump the revision of this structure.

Reply:
But it is declared as an array ...
`PG_DECLARE_ARRAY(gyroDeviceConfig_t, MAX_GYRODEV_COUNT, gyroDeviceConfig);`

Reply:
See https://github.com/betaflight/betaflight/blob/a6d70be96a4c3492c56fff7904086152f6875e91/src/main/pg/gyrodev.c#L94